exponent_server_sdk/__init__.py

```diff
@@ -335,8 +342,12 @@ def _publish_internal(self, push_messages):
             push_messages: An array of PushMessage objects.
         """
 
+        url = self.host + self.api_url + '/push/send'
+        if self.force_fcm_v1:
+            url += '?useFcmV1=true'
```

Comment:
1. Prefer to use urllib to construct the URL
2. Would be good to have logic that sets useFcmV1=true/false if self.force_fcm_v1 is True/False, and omits the query param if the Python option is None

Reply:
@ide did you think about something like that?